_posts/2020-10-01-annual-report-part-2.md

```diff
@@ -0,0 +1,49 @@
+
+## Cohort expense and cost - and its shoestring budget 
+
+Our website helps to give us an incredibly professional face thanks to Bérénice’s tireless and utterly incredible efforts (including during her maternity leave). But, as shiny as our site might look, we’re still operating on a shoestring budget - most of the preparation work we do is out of hours and on our own time. Likewise, OLS mentors and experts are all currently unpaid. 
+
+Estimated real cost for each cohort is GBP £24,259 in founder, mentor, and expert time, as well as some infrastructure costs. Ideally, we would like to meet these costs - as such it probably comes as no surprise that the OLS team is currently actively seeking funding and applying for external fund sources.
+
+<iframe src="https://docs.google.com/spreadsheets/d/1_Q8atU9Xsh5aXQuQpxizyMUdKWug_jYgMobzhxxPE8c/pubhtml?widget=true&amp;headers=false"></iframe>
+
+*OLS real cost for one cohort (financial requirements are currently not met)*
+
+Many of the expenses required to run this project effectively have come from the founders’ own pockets. Some funding resources can be reached with requests for support to sympathetic organisations, however claiming some of our expenses can sometimes be challenging -- not all items can be funded due to funding organisation's policies, and administrative formalities and/or blog posts are required for each funded item. Whilst running a non-profit organisation outside of our day jobs, we’ve unfortunately become extremely time-poor to constantly look for small funds or offer invisible labour associated with funding obligations.
+
+### Software Sustainability Institute partnership for OLS-2
+
+Two co-founders are fellows of the Software Sustainability Institute (UK), which provided small fellowship funds with which we manage to support the communication and application management platforms required for OLS-2 (EasyChair, Zoom, professional mentor training etc.). We are grateful to receive personal and organisational support in redirecting this fellowship towards OLS infrastructure needs. 
+
+### EOSC-Life Training grant to partially support OLS-3
+
+For one cohort in 2021, OLS has been awarded EOSC-Life (https://www.eosc-life.eu/) open call training funding of up to 20,000 Euros to develop a special call to train and mentor up to 15 projects from within the EOSC-Life research infrastructure. This will partially cover costs for OLS-3, which will be designed over the next few months by including a new curriculum.
```

Comment:
```suggestion
For one cohort in 2021, OLS has been awarded [EOSC-Life](https://www.eosc-life.eu/) open call training funding of up to 20,000€ to develop a special call to train and mentor up to 15 projects from within the EOSC-Life research infrastructure. This will partially cover costs for OLS-3, which will be designed over the next few months by including a new curriculum.
```